framework/db/ActiveRecord.php

```diff
@@ -268,7 +268,7 @@ public static function deleteAll($condition = '', $params = [])
      */
     public static function find()
     {
-        return Yii::createObject(ActiveQuery::className(), [get_called_class()]);
+        return Yii::createObject(ActiveQuery::class, [get_called_class()]);
```

Comment:
> since it could be inherited from ActiveQuery.

I think you meant `ActiveRecord`. And if thats the case then `static::class` will work perfectly.

if you mean `ActiveQuery` when I must ask for an example on what that is.